api/components/image.schema.json

```diff
@@ -12,16 +12,71 @@
             ]
         },
         "path": {
-            "description": "The image path from the resources folder",
+            "description": "The image path from the resources folder or from the network depending on the value of fromNetwork.",
             "type": "string"
         },
+        "fromNetwork": {
+            "description": "Whether the image will be loaded from a network url of from an application's resource path.",
+            "type": "boolean",
+            "default": false
+        },
         "width": {
-            "description": "The image width",
+            "description": "The image width.",
             "type": "number"
         },
         "height": {
-            "description": "The image height",
+            "description": "The image height.",
             "type": "number"
+        },
+        "alignment": {
+            "description": "How to align the image.",
+            "$ref": "defs/alignment.schema.json"
+        },
+        "centerSlice": {
+            "description": "The center slice for a nine-patch image.",
```

Comment:
This description is unclear to me. What is this property actually do ?

Reply:
It is unclear because it needs a bit of background, you need to know what is a nine-patch image to understand this property.